addons/html_editor/static/tests/insert/line_break.test.js

```diff
@@ -190,12 +190,8 @@ describe("Selection collapsed", () => {
             await testEditor({
                 contentBefore: "<p><b>abc[]</b> def</p>",
                 stepFunction: insertLineBreak,
-                // The space is converted to a non-breaking space so
-                // it is visible (because it's after a <br>).
-                // Visually, the caret does show _after_ the line
-                // break.
                 // JW cAfter: '<p><b>abc[]<br></b>&nbsp;def</p>',
-                contentAfter: "<p><b>abc<br>[]</b>&nbsp;def</p>",
+                contentAfter: "<p><b>abc<br>[]</b> def</p>",
```

Comment:
The space before def is not rendered if it is not a `&nbsp;`, which means when you press enter before " def", it looks like the space is removed.

Reply:
Done!